### PR TITLE
content/en/docs/16.0: document backup/restore metrics

### PR DESCRIPTION
**Pull request by maxenglander**
Document existing backup/restore metrics.

### CODE REVIEW

**Comment by deepthi** [comment]
@maxenglander given that we are deferring https://github.com/vitessio/vitess/pull/11979 to v17, can you do a smaller PR that documents just the existing metrics? That would be much appreciated.

**Comment by maxenglander** [comment]
@deepthi 

> @maxenglander given that we are deferring [vitessio/vitess#11979](https://github.com/vitessio/vitess/pull/11979) to v17, can you do a smaller PR that documents just the existing metrics? That would be much appreciated.

Can do!

File: content/en/docs/16.0/reference/compatibility/_index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Compatibility
 description: Reference documents for MySQL compatibility with Vitess 
-weight: 4
+weight: 5
```
Comment (maxenglander) [inline content/en/docs/16.0/reference/compatibility/_index.md:4]: The rest of the changes in this PR are to preserve the current ordering of pages, and to ensure the new page added in this PR is at the bottom of the list.

**Review by rsajwani — APPROVED** [review]
LGTM

**Pull request merged**